numba_cuda/numba/cuda/core/interpreter.py

```diff
@@ -1437,7 +1440,7 @@ def interpret(self, bytecode):
         if PYVERSION in ((3, 10), (3, 11), (3, 12), (3, 13)):
             peepholes.append(peep_hole_list_to_tuple)
         peepholes.append(peep_hole_delete_with_exit)
-        if PYVERSION in ((3, 10), (3, 11), (3, 12), (3, 13)):
+        if PYVERSION in ((3, 9), (3, 10), (3, 11), (3, 12), (3, 13)):
```

Comment:
I think maybe this should have not been added here, but the check above - it seems that `peep_hole_list_to_tuple` is the peephole that pertains also to 3.9.

For reference, I am looking at the interpreter.py implementation from Numba 0.60.0:

https://github.com/numba/numba/blob/53e976f1b0c6683933fa0a93738362914bffc1cd/numba/core/interpreter.py#L1421-L1432

Reply:
Great catch, thank you for finding this!